examples/java/java-sin/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.wandbj.sinfunc</groupId>
```

Comment:
Is there a reason for this strange wandbj identifier? should we call this something like:
com.wandb.examples.sinfunc
?

Reply:
or com.wandb.java.examples? or something cleaner?

Reply:
Using 'wandb' would cause to folder to be ignored by git, since this is the name of the folder that wandb also generates. Since it is not always clear where the user maybe running the command from i changed the name to wandbj so it is explicit that this folder is not generated.

I can change it to wandb and add custom conditions to .gitignore.

Reply:
If I didn't make it clear: the packageId should match the folder structure in src/main/java. @raubitsj What do you want to do about this? Do you want me to change it or can I leave it as wandbj

Reply:
i guess i dont care that much... I just think if this is the first example we should make it as clean as java lets us make it.

Reply:
I felt this it is pretty clear since the groupId doesn't really matter when developing packages locally. I'm going to close this issue for now. We can always change it later if needed